<?xml version="1.0" encoding="utf-8"?>
<launch>
    <!-- transform from map to world -->
    <node pkg="tf" type="static_transform_publisher" name="world_to_map_broadcaster" args="0 0 0 0 0 0 1 world map 100" />

    <!-- global pose of spheros -->
    <node ns="sphero_rgw" name="global_pose" pkg="hri_sphero_test" type="global_pose.py" args="--robot_start_pose '[0.5,1.6,0]' --robot_frame sphero_rgw" output="screen"/>
    <node ns="sphero_wrb" name="global_pose" pkg="hri_sphero_test" type="global_pose.py" args="--robot_start_pose '[2.5,1.4,0]' --robot_frame sphero_wrb" output="screen"/>
    <node ns="sphero_yoo" name="global_pose" pkg="hri_sphero_test" type="global_pose.py" args="--robot_start_pose '[0.8,2.5,0]' --robot_frame sphero_yoo" output="screen"/>
    <node ns="sphero_wpp" name="global_pose" pkg="hri_sphero_test" type="global_pose.py" args="--robot_start_pose '[2.0,0.5,0]' --robot_frame sphero_wpp" output="screen"/>

    <node ns="sphero_ggw" name="global_pose" pkg="hri_sphero_test" type="global_pose.py" args="--robot_start_pose '[0.5,0.5,0]' --robot_frame GPSReceiverHelmet_goodaxes" output="screen"/>

    <!-- social force algorithm -->
    <node name="social_force" pkg="hri_sphero_test" type="social_force.py" output="screen"/>

</launch>
